### ISSUE TITLE
DeepLabCut 2.3.5 GUI / PySide6  - AttributeError: 'PySide6.QtGui.QSurfaceFormat' object has no attribute 'DoubleBuffer'

### ISSUE THREAD

**Issue by ovalerio** [DeepLabCut/DeepLabCut#2313]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Bug description

Created a new project using DLC GUI.  The `Label Frames` step is showing an error message and the `napari` viewer is not opening:
```
 $ python3
Python 3.8.17 | packaged by conda-forge | (default, Jun 16 2023, 07:06:00) 
[GCC 11.4.0] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> import deeplabcut
2023-07-17 17:55:30.352677: I tensorflow/core/util/port.cc:110] oneDNN custom operations are on. You may see slightly different numerical results due to floating-point round-off errors from different computation orders. To turn them off, set the environment variable `TF_ENABLE_ONEDNN_OPTS=0`.
2023-07-17 17:55:30.417672: I tensorflow/core/platform/cpu_feature_guard.cc:182] This TensorFlow binary is optimized to use available CPU instructions in performance-critical operations.
To enable the following instructions: AVX2 AVX512F AVX512_VNNI FMA, in other operations, rebuild TensorFlow with the appropriate compiler flags.
2023-07-17 17:55:32.956687: W tensorflow/compiler/tf2tensorrt/utils/py_utils.cc:38] TF-TRT Warning: Could not find TensorRT
Loading DLC 2.3.5...
>>> deeplabcut.launch_dlc()
14636it [03:42, 65.92it/s]
/home/valerio/anaconda3/envs/dlc/lib/python3.8/site-packages/sklearn/cluster/_kmeans.py:1930: FutureWarning: The default value of `n_init` will change from 3 to 'auto' in 1.4. Set the value of `n_init` explicitly to suppress the warning
  super()._check_params_vs_input(X, default_n_init=3)
Traceback (most recent call last):
  File "/home/valerio/anaconda3/envs/dlc/lib/python3.8/site-packages/deeplabcut/gui/tabs/label_frames.py", line 55, in label_frames
    _ = launch_napari(folder)
  File "/home/valerio/anaconda3/envs/dlc/lib/python3.8/site-packages/deeplabcut/gui/widgets.py", line 38, in launch_napari
    viewer = napari.Viewer()
  File "/home/valerio/anaconda3/envs/dlc/lib/python3.8/site-packages/napari/viewer.py", line 67, in __init__
    self._window = Window(self, show=show)
  File "/home/valerio/anaconda3/envs/dlc/lib/python3.8/site-packages/napari/_qt/qt_main_window.py", line 463, in __init__
    self._qt_window = _QtMainWindow(viewer)
  File "/home/valerio/anaconda3/envs/dlc/lib/python3.8/site-packages/napari/_qt/qt_main_window.py", line 88, in __init__
    self._qt_viewer = QtViewer(viewer, show_welcome_screen=True)
  File "/home/valerio/anaconda3/envs/dlc/lib/python3.8/site-packages/napari/_qt/qt_viewer.py", line 217, in __init__
    self._create_canvas()
  File "/home/valerio/anaconda3/envs/dlc/lib/python3.8/site-packages/napari/_qt/qt_viewer.py", line 401, in _create_canvas
    self.canvas = VispyCanvas(
  File "/home/valerio/anaconda3/envs/dlc/lib/python3.8/site-packages/napari/_vispy/canvas.py", line 33, in __init__
    super().__init__(*args, **kwargs)
  File "/home/valerio/anaconda3/envs/dlc/lib/python3.8/site-packages/vispy/scene/canvas.py", line 135, in __init__
    super(SceneCanvas, self).__init__(
  File "/home/valerio/anaconda3/envs/dlc/lib/python3.8/site-packages/vispy/app/canvas.py", line 211, in __init__
    self.create_native()
  File "/home/valerio/anaconda3/envs/dlc/lib/python3.8/site-packages/vispy/app/canvas.py", line 228, in create_native
    self._app.backend_module.CanvasBackend(self, **self._backend_kwargs)
  File "/home/valerio/anaconda3/envs/dlc/lib/python3.8/site-packages/vispy/app/backends/_qt.py", line 372, in __init__
    self._init_specific(p, kwargs)
  File "/home/valerio/anaconda3/envs/dlc/lib/python3.8/site-packages/vispy/app/backends/_qt.py", line 791, in _init_specific
    glformat = _set_config(p.context.config)
  File "/home/valerio/anaconda3/envs/dlc/lib/python3.8/site-packages/vispy/app/backends/_qt.py", line 283, in _set_config
    glformat.setSwapBehavior(glformat.DoubleBuffer if c['double_buffer']
AttributeError: 'PySide6.QtGui.QSurfaceFormat' object has no attribute 'DoubleBuffer'
``` 

### Operating System

NAME="CentOS Linux"
VERSION="7 (Core)"
ID="centos"
ID_LIKE="rhel fedora"
VERSION_ID="7"
PRETTY_NAME="CentOS Linux 7 (Core)"

### DeepLabCut version

DeepLabCut 2.3.5

### DeepLabCut mode

single animal

### Device type

```
+-----------------------------------------------------------------------------+
| NVIDIA-SMI 525.85.12    Driver Version: 525.85.12    CUDA Version: 12.0     |
|-------------------------------+----------------------+----------------------+
| GPU  Name        Persistence-M| Bus-Id        Disp.A | Volatile Uncorr. ECC |
| Fan  Temp  Perf  Pwr:Usage/Cap|         Memory-Usage | GPU-Util  Compute M. |
|                               |                      |               MIG M. |
|===============================+======================+======================|
|   0  Quadro RTX 6000     On   | 00000000:2F:00.0 Off |                    0 |
| N/A   25C    P8    12W / 250W |      3MiB / 23040MiB |      0%      Default |
|                               |                      |                  N/A |
+-------------------------------+----------------------+----------------------+
```

### Anything else?

I found a related bug report in the image.sc website.

https://forum.image.sc/t/attributeerror-after-calling-napari-viewer/78270 

_No response_

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/DeepLabCut/DeepLabCut/blob/master/CODE_OF_CONDUCT.md)

**Comment by n-poulsen** [comment]
Hi @ovalerio, could you please report the napari version that you're using? You can do so by reporting the output of `napari --info` (from your conda environment). The output should look something like this:
```
napari: 0.4.17
Platform: ...
System: ...
Python: ...
Qt: 6.3.1
PySide6: 
NumPy: 1.23.5
SciPy: 1.10.1
Dask: 2023.7.0
VisPy: 0.11.0
magicgui: 0.7.2
superqt: unknown
in-n-out: 0.1.8
app-model: 0.2.0
npe2: 0.7.1

...

Plugins:
  - napari: 0.4.17 (77 contributions)
  - napari-console: 0.0.8 (0 contributions)
  - napari-deeplabcut: 0.1.2 (16 contributions)
  - napari-svg: 0.1.10 (2 contributions)
```

**Comment by jeylau** [comment]
Will be solved with `napari-deeplabcut==0.2.0`. The release is coming soon, we'll let you know 😊

**Comment by jeylau** [comment]
@ovalerio, the new release is now online, you can `pip install napari-deeplabcut==0.2.0` 😊
